docker/policies/add_instead_of_copy_test.rego

```diff
@@ -0,0 +1,37 @@
+package appshield.DS005
+
+test_failAdd_basic {
+	not failAdd with input as  {"stages": {"foo":[{"Cmd": "run", "Value": ["tar -xjf /temp/package.file.tar.gz"]}]}}
+}
+
+test_failAdd_copy {
+	not failAdd with input as {"stages": {"foo":[{"Cmd": "copy", "Value": ["test.txt", "test2.txt"]}]}}
+}
+
+test_failAdd_add {
+	failAdd with input as {"stages": {"foo":[{"Cmd": "add", "Value": ["/target/app.jar", "app.jar"]}]}}
+}
+
+test_failAdd_many_add {
+	failAdd with input as {"stages": {"foo":[
+		{"Cmd": "add", "Value": ["/target/resources.tar.gz", "resources.jar"]},
+		{"Cmd": "add", "Value": ["/target/app.jar", "app.jar"]},
+	]}}
+}
+
+test_failAdd_add_archive {
+	not failAdd with input as {"stages": {"foo":[{"Cmd": "add", "Value": ["/target/resources.tar.gz", "resources.jar"]}]}}
+}
+
+test_entry_point_positive {
```

Comment:
As mentioned above, we always test only `deny`, so the test rule name should describe the case more.

```suggestion
test_add_non_tar_positive {
```